mapreduce/tests/test_integration.py

```diff
@@ -0,0 +1,18 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from . import common
+
+pytestmark = pytest.mark.integration
```

Comment:
Is there a reason why this line is needed ?

Reply:
This is needed for separating the e2e tests and the unit tests in different environment (like here https://github.com/DataDog/integrations-core/blob/82c1b4e0319b30957e2a222d0e681a6b25a5aeca/hdfs_datanode/tox.ini#L21)